src/scss/_variables.scss

```diff
@@ -1,11 +1,12 @@
 // Default border radius
-$border-radius: 4px !default;
+$border-radius: 8px !default;
 
 // Default spacing
 $spacing: 2rem !default;
 
 // Brand color
 $c-brand: #ffbd13 !default;
+$c-brand-tx: #fff !default;
```

Comment:
The "tx" suffic should be discussed.
If you see below, "fg" is used for foreground.